tests/unit/helpers/validate-properties-test.js

```diff
@@ -26,3 +28,58 @@ testInvalidPropertyValues('baz', ['', null, undefined], function(subject) {
 testValidPropertyValues('baz', ['Winston', '12345', null, undefined, ''], function(subject) {
   subject.set('isBaz', false);
 });
+
+moduleFor('controller:foo', 'Unit - Ensure validate properties test helpers fail when invalid', {
+  needs: [
+    'ember-validations@validator:local/presence',
+    'ember-validations@validator:local/length'
+  ],
+
+  /* jshint ignore:start */
+  beforeEach: function() {
+    // use inverse of deepEqual to ensure the test helpers fail when invalid
+    _deepEqual = deepEqual;
```

Comment:
It doesn't seem like I can add `assert` as an argument to `beforeEach` (it blows up). But I can just use `QUnit.assert` in the `beforeEach` and that works. Is that what you mean?